packages/atomic/src/pages/examples/commerce-website/listing-surf-accessories.html

```diff
@@ -28,37 +26,6 @@ <h1>Product listing page</h1>
       <section>
         <atomic-commerce-interface type="product-listing">
           <atomic-commerce-product-list display="grid" density="compact" image-size="small">
-            <atomic-product-template>
-              <template>
-                <atomic-product-image
-                  class="thumbnail"
-                  field="ec_thumbnails"
-                  fallback="https://picsum.photos/350"
-                ></atomic-product-image>
-
-                <atomic-product-link class="font-bold"></atomic-product-link>
-
-                <atomic-field-condition class="brand text-neutral-dark" if-defined="ec_brand">
-                  <atomic-product-text field="ec_brand"></atomic-product-text>
-                </atomic-field-condition>
-
-                <atomic-field-condition class="field" if-defined="ec_rating">
-                  <atomic-result-rating field="ec_rating"></atomic-result-rating>
-                </atomic-field-condition>
-
-                <atomic-field-condition class="mt-4 text-2xl font-bold field" if-defined="ec_price">
-                  <atomic-result-number field="ec_price">
-                    <atomic-format-currency currency="CAD"></atomic-format-currency>
-                  </atomic-result-number>
-                </atomic-field-condition>
-                <atomic-product-description></atomic-product-description>
-                <atomic-result-fields-list>
-                  <atomic-field-condition must-match-ec_in_stock="true">
-                    <atomic-result-badge label="In stock"></atomic-result-badge>
-                  </atomic-field-condition>
-                </atomic-result-fields-list>
-              </template>
-            </atomic-product-template>
```

Comment:
Using the default template here to simplify keeping these up to date with the new template components